sig-robotics/examples/Implement-VSLAM-algorithm-based-on-KubeEdge/README.md

```diff
@@ -0,0 +1,116 @@
+# Implement VSLAM algorithm based on KubeEdge
+
+This project aims to develop a Visiual SLAM application on an edge-side robot and implementing compute offloading to reduce the compute and memory resource pressure on edge-side.
+
+## Prerequisites
+
+1. Kubeedge  1.10.1+
+2. Edgemesh  1.11.0+
+
+## Steps to reproduce
+
+1. Make sure your cluster is set up and [edgemesh](https://kubeedge.io/en/docs/advanced/edgemesh/) is enabled.
+
+2. Clone the community/sig-robotics respository.
+
+   ```bash
+   git clone https://github.com/kubeedge/community
+   ```
+
+3. Prepare your RGBD dataset or RGBD-sensors on the edge-side.
+
+   > We use [TUM Datasets](https://vision.in.tum.de/data/datasets/rgbd-dataset/download) and its format in our development and test process.
+
+4. Check your work node's name and modify the deployment.yaml file.
+
+   > For example, we have a master node and two work nodes, named edge01 and edge02
+
+   `cloud-deployment.yaml`
+
+   ```yaml
+   spec:
+     template:
+       spec:
+         nodeName: edge02
+   ```
+
+   `edge-deployment.yaml`
+
+   ```yaml
+   spec:
+     template:
+       spec:
+         nodeName: edge01
+   ```
+
+   Check the location of the downloaded  [TUM Datasets](https://vision.in.tum.de/data/datasets/rgbd-dataset/download) and modify `edge-deployment.yaml`
```

Comment:
download [TUM Datasets](https://vision.in.tum.de/data/datasets/rgbd-dataset/download) to your edge node, and mount it within k8s.